readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -79,7 +79,7 @@ context = {
     'current_version': "{{ version.verbose_name }}",
     'version_slug': "{{ version.slug }}",
     'MEDIA_URL': "{{ settings.MEDIA_URL }}",
-    'STATIC_URL': "{{ settings.STATIC_URL }}",
+    'STATIC_URL': "{{ proxied_static_path }}",
```

Comment:
I'm replacing this value, but we could also introduce a new one (it would require to change our extension)

Reply:
I'd vote for adding a new variable. There could be something random depending on this, and I always prefer adding something new rather than changing something in a backwards incompatible way.